src/views/PageNotFound.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { useEffect } from "react";
+import { useNavigate } from "react-router-dom";
+
+export function PageNotFound() {
+	const navigate = useNavigate();
+
+	useEffect(() => {
+		const timer = setTimeout(() => {
+			navigate("/", { replace: true });
+		}, 3000);
+
+		return () => clearTimeout(timer);
```

Comment:
Nice touch with the use of `navigate` here, and great job thinking about clearing the timeout!